contracts/integrations/src/deployment_manager.ts

```diff
@@ -9,13 +9,7 @@ import {
 import { artifacts as ERC1155Artifacts, ERC1155MintableContract } from '@0x/contracts-erc1155';
```

Comment:
Why is this file being exported? Isn't `DeploymentManager` only used by the tests in this package?

Reply:
Good catch, reverted this change.